lib/rake-tasks-docker/rake_task.rb

```diff
@@ -1,37 +1,105 @@
 
+require 'slop'
 require 'timeout'
+require 'highline'
 
 require_relative 'services'
 
 namespace :docker do
-  task :status, :services do |task, args|
-    services = RakeTasksDocker::Services.new(args[:services] ? args[:services].split(' ') : [])
-    puts services.status
+  def parse_argv
+    args = ARGV
+    args = ARGV[2..-1] if ARGV.length > 2
+    args
+  end
+
+  def parse_options(args = [])
+    args = parse_argv if args.empty?
+    Slop.parse args do |options|
+      options.array '-s', '--services', 'The docker services to interact with, separated by comma', default: []
+      yield options if block_given?
+    end
+  end
```

Comment:
is it possible we can stay with arguments instead of options, as they map better to how docker-compose does things?